mm2src/mm2_test_helpers/src/for_tests.rs

```diff
@@ -872,15 +870,12 @@ impl Drop for RaiiDump {
         let log = String::from_utf8_lossy(&log);
         let log = log.trim();
 
-        if let (true, true, mut stdout) = (nocapture, *ISATTY, std::io::stdout()) {
-            execute!(
-                stdout,
-                SetForegroundColor(Color::DarkYellow),
-                Print(format!("vvv {:?} vvv\n", self.log_path)),
-                SetForegroundColor(Color::Yellow),
-                Print(log),
-            )
-            .expect("Printing to stdout failed");
+        // If we want to determine is a tty or not here and write logs to stdout only if it's tty,
+        // we can use something like https://docs.rs/atty/latest/atty/ here, look like it's more cross-platform than gstuff::ISATTY .
+
+        if nocapture == true {
+            std::io::stdout().write(format!("vvv {:?} vvv\n", self.log_path).as_bytes()).expect("Printing to stdout failed");
+            std::io::stdout().write(log.as_bytes()).expect("Printing to stdout failed");
```

Comment:
for better compatibility:

```suggestion
        std::io::stdout()
            .write(format!("{}vvv {:?} vvv\n", DARK_YELLOW_ANSI_CODE, self.log_path).as_bytes())
            .expect("Printing to stdout failed");
        std::io::stdout()
            .write(format!("{}{}{}", YELLOW_ANSI_CODE, log, RESET_COLOR_ANSI_CODE).as_bytes())
            .expect("Printing to stdout failed");
```

and create following consts inside of the function:
```rs
const DARK_YELLOW_ANSI_CODE: &str = "\x1b[33m";
const YELLOW_ANSI_CODE: &str = "\x1b[93m";
const RESET_COLOR_ANSI_CODE: &str = "\x1b[0m";
```